File: Backend/RoomMe/RoomMe.API/Models/SignInReturnModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RoomMe.API.Models
{
    public class SignInReturnModel
    {
        public int Id { get; set; }
        public string Nickname { get; set; }
        public string Email { get; set; }
        public string Firstname { get; set; }
        public string Lastname { get; set; }
        public string PhoneNumber { get; set; }
        public string AccessToken { get; set; }
        public string RefreshToken { get; set; }
    }
}

File: Android-app/domain/src/main/java/uj/roomme/domain/auth/SignInReturnModel.kt
package uj.roomme.domain.auth

data class SignInReturnModel(
    val id: Int,
    val nickname: String,
    val email: String,
    val firstname: String?,
    val lastname: String?,
    val phoneNumber: String?,
    val accessToken: String,
    val refreshToken: String
)